toolchain_containers.bzl

```diff
@@ -0,0 +1,20 @@
+def toolchain_container_sha256s():
```

Comment:
Let's avoid top-level skylark code. Could you move this file into `rules` ?

Reply:
Sure.